src/components/register/RegisterForm.vue

```diff
@@ -0,0 +1,250 @@
+<template>
+    <b-form v-on:submit.prevent="onSubmit" class="register">
+        <h1 class="field_title">Create my account</h1>
+        <div class="field">
+            <label class="field__label" for="first_name">First name</label>
+            <b-form-input
+                id="first_name"
+                type="name"
+                placeholder=" "
+                autocomplete=" "
+                aria-describedby="first_name-constraints"
+                aria-required="true"
+                class="field__input"
+                @focus="resetError()"
+                @blur="check('first_name')"
+                v-model="name.value"
+            />
+        </div>
+        <div class="field">
+            <label class="field__label" for="last_name">Last name</label>
+            <b-form-input
+                id="first_name"
```

Comment:
```suggestion
                id="last_name"
```
Un petit oubli je pense, mais du coup le for et l'id ne correspondent pas.